### DISCUSSION TITLE
Feature Request: db:import --latest

### DISCUSSION THREAD

**Discussion by ChristianOsswald** [sourcebroker/deployer-extended-database#29] [Q&A] [answered]
Hello,

I would like to import the latest db dump. Would it be a big thing, or could you easily integrate that functionality if it isn't already existing?

Kind regards
Christian

**Comment by kszymukowicz** [comment, marked as answer]
This is possible now in v18.
To get last dump use `fromLocalStorage` option like:
```
dep db:pull production --options=fromLocalStorage
```
